src-plugins/mscAlgorithmPaint/mscAlgorithmPaintToolBox.cpp

```diff
@@ -1257,7 +1257,7 @@ void AlgorithmPaintToolBox::updateMouseInteraction() //Apply the current interac
 dtkPlugin* AlgorithmPaintToolBox::plugin()
 {
     medPluginManager* pm = medPluginManager::instance();
-    dtkPlugin* plugin = pm->plugin ( "mscAlgorithmPaint" );
+    dtkPlugin* plugin = pm->plugin ( "Algorithm Paint" );
```

Comment:
After more work on that, this name is the same as the name of the information widget, so i standardize also that putting a name understandable by users and dev, as this one